src/features/RendererRegistrar.cpp

```diff
@@ -19,17 +20,23 @@ namespace Features
 
         static SceneToRenderersMap renderers;
         INIT_RENDERER_MAP(renderers)
+			
+#ifdef _TEST
```

Comment:
Can you help correcting the formatting here?

Reply:
I did it based on my understanding. I might not be done according to your perspective. 
Thanks